test/e2e/policies.test-suite/topology-aware/n4c16/test00-reject-symlink/code.var.sh

```diff
@@ -0,0 +1,48 @@
+# This test verifies that nri-resmgr properly refuses to start if a cache
```

Comment:
better to name the test to something else than "test00" as there is already a test00, better to use test13 (next available free number)